crates/fuzzing/wasm-spec-interpreter/build.rs

```diff
@@ -0,0 +1,85 @@
+/// Build the OCaml code and statically link it into the Rust library; see the
+/// [ocaml-interop
+/// example](https://github.com/tezedge/ocaml-interop/blob/master/testing/rust-caller/build.rs)
+/// for more details. After playing with this a bit, I discovered that the best
+/// approach to avoid missing symbols was to imitate `dune`: I observed `rm -rf
+/// _build && dune build ./ocaml/interpret.exe.o --display=verbose` and used
+/// that as a pattern, now encoded in `ocaml/Makefile` for easier debugging.
+use std::{env, path::PathBuf, process::Command};
+
+const LIB_NAME: &'static str = "interpret";
+const OCAML_DIR: &'static str = "ocaml";
+const SPEC_DIR: &'static str = "ocaml/spec";
```

Comment:
I don't know if you ever want to publish this crate, but if you do, this will need to be moved to the `OUT_DIR`. Cargo refuses to publish crates that modify their source directory when building.

Reply:
This crate is marked `publish = false`.